python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -415,14 +415,20 @@ def _convert_yaml_path_to_absolute_path(self):
                                 self._dataset_dir, data["path"]
                             )
 
-    def load(self):
+    def load(self, selected_task: list = None):
         """Load the dataset."""
```

Comment:
add detailed docstring including parameters so that user could figure out how to load task selectively.

Reply:
For example, `we're loading via task name .....`